tools/vulnerability-scan/wss-scan.config

```
@@ -1,3 +1,3 @@
 baseDirPath=$(pwd)
 gitBasePath=https://github.com/opensearch-project/
-gitRepos=OpenSearch,OpenSearch-Dashboards,alerting,alerting-dashboards-plugin,anomaly-detection,anomaly-detection-dashboards-plugin,asynchronous-search,common-utils,cross-cluster-replication,dashboards-reports,dashboards-visualizations,data-prepper,index-management,index-management-dashboards-plugin,job-scheduler,k-NN,opensearch-java,opensearch-js,opensearch-py,opensearch-dsl-py,performance-analyzer,performance-analyzer-rca,perftop,security,security-dashboards-plugin,sql,observability,opensearch-build,ml-commons,notifications,geospatial,dashboards-maps
+gitRepos=OpenSearch,OpenSearch-Dashboards,alerting,alerting-dashboards-plugin,anomaly-detection,anomaly-detection-dashboards-plugin,asynchronous-search,common-utils,cross-cluster-replication,dashboards-reports,dashboards-visualizations,data-prepper,index-management,index-management-dashboards-plugin,job-scheduler,k-NN,opensearch-java,opensearch-js,opensearch-py,opensearch-dsl-py,performance-analyzer,performance-analyzer-rca,perftop,security,security-dashboards-plugin,sql,observability,opensearch-build,ml-commons,notifications,geospatial,dashboards-map,neural-search,security-analytics,security-analytics-dashboards-plugin,dashboards-search-relevance,opensearch-migrations
```

Its dashboard-maps https://github.com/opensearch-project/dashboards-maps
typo here
```suggestion
gitRepos=OpenSearch,OpenSearch-Dashboards,alerting,alerting-dashboards-plugin,anomaly-detection,anomaly-detection-dashboards-plugin,asynchronous-search,common-utils,cross-cluster-replication,dashboards-reports,dashboards-visualizations,data-prepper,index-management,index-management-dashboards-plugin,job-scheduler,k-NN,opensearch-java,opensearch-js,opensearch-py,opensearch-dsl-py,performance-analyzer,performance-analyzer-rca,perftop,security,security-dashboards-plugin,sql,observability,opensearch-build,ml-commons,notifications,geospatial,dashboards-maps,neural-search,security-analytics,security-analytics-dashboards-plugin,dashboards-search-relevance,opensearch-migrations
```

---

Thanks.